_resources/port1.0/group/gnuradio-1.0.tcl

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8; mode: tcl; c-basic-offset: 4; indent-tabs-mode: nil; tab-width: 4; truncate-lines: t -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+#
+# This PortGroup helps to maintain gnuradio dependencies coherent between
+# gnuradio and gnuradio modules.
+#
+# Usage:
+#
+# PortGroup           gnuradio 1.0
+#
+
+PortGroup           cmake 1.1
+PortGroup           boost 1.0
+
+categories          science comms
+platforms           darwin macosx
+
+# declare which gnuradio are you using
+options gnuradio.type
+default gnuradio.type "gnuradio"
+
+options gnuradio.python_version
+options gnuradio.default_python_variant
+
+if {[string first "37" $subport] > 1} {
+    default gnuradio.python_versions { 2.7 }
+    default gnuradio.default_python_variant +python27
+} else {
+    # the following versions use all the same python
+    default gnuradio.python_versions { 3.8 3.9 3.10 }
+    default gnuradio.default_python_variant +python39
+}
+
+# use C/C++11
+compiler.c_standard   2011
+compiler.cxx_standard 2011
+boost.version 1.71
+
+# see https://github.com/macports/macports-ports/pull/7805
+license_noconflict-append openssl
+
+# see https://github.com/macports/macports-ports/pull/7805
+license_noconflict-append graphviz
+
+# Define the available variants
+foreach py_ver ${gnuradio.python_versions} {
+    set py_ver_no_dot [join [split ${py_ver} "."] ""]
+    set variant_line {variant python${py_ver_no_dot} description "Build with python ${py_ver} support"}
+    foreach py_over ${gnuradio.python_versions} {
+        if { ${py_ver} == ${py_over} } { continue }
+        set py_over_no_dot [join [split ${py_over} "."] ""]
+        append variant_line " conflicts python${py_over_no_dot}"
+    }
+    append variant_line { { } }
+    eval $variant_line
+    if {[variant_isset python${py_ver_no_dot}]} {
+        if {${gnuradio.default_python_variant} != "+python${py_ver_no_dot}"} {
+            set gnuradio.default_python_variant ""
+        }
+    }
+}
+
+# set default python variant if not selected
+if {${gnuradio.default_python_variant} != ""} {
+    default_variants-append "${gnuradio.default_python_variant}"
+}
+
+# If a python variant is enabled, activate it
+set active_python_version ""
+set active_python_version_no_dot ""
+foreach py_ver ${gnuradio.python_versions} {
+    set py_ver_no_dot [join [split ${py_ver} "."] ""]
+    if {[variant_isset python${py_ver_no_dot}]} {
+        set active_python_version        ${py_ver}
+        set active_python_version_no_dot ${py_ver_no_dot}
+    }
+}
+
+depends_build-append \
+    port:pkgconfig
+
+depends_lib-append \
+    port:fftw-3-single \
```

Comment:
I think these next 4 dependencies (fftw though gmp) belong in the GR Portfile, not here, since they are not common to all GR OOT modules.

Reply:
Yes I was quite lazy with dependencies since:
- I see that sometimes few of those (like ffwt, mpir and volk) was declared also in gr modules
- since are lib they don't create too much hassle

therefore, I prefer to leave as is since it doesn't create false dependency installation.